release/pkg/assets_package_controller.go

```diff
@@ -64,27 +66,49 @@ func (r *ReleaseConfig) GetPackagesAssets() ([]Artifact, error) {
 		ProjectPath:       packagesRootPath,
 		SourcedFromBranch: sourcedFromBranch,
 	}
+	// Remove the OS, and Arch for the Helm chart, and modify the tag based off build-tooling helm/push.sh script.
+	// Currently we must Trim leading v of the tag after the :, and add "-helm to the end of tag for source URI.
+	// 123456.dkr.ecr.us-west-2.amazonaws.com/eks-anywhere-packages:v0.1.2-2e9994fd1afb2216a51fa474ac5d7dc6a772bb62
+	// >>>
+	// 123456.dkr.ecr.us-west-2.amazonaws.com/eks-anywhere-packages:0.1.2-2e9994fd1afb2216a51fa474ac5d7dc6a772bb62-helm
 	artifacts := []Artifact{{Image: imageArtifact}}
+	helmImageArtifact := &ImageArtifact{
+		AssetName:         packagesHelmChart, // This needs to differ from the image name for later steps
+		SourceImageURI:    strings.ReplaceAll(fmt.Sprintf("%s-helm", sourceImageUri), "packages:v", "packages:"),
+		ReleaseImageURI:   strings.ReplaceAll(releaseImageUri, "packages:v", "packages:"),
```

Comment:
The release URI won't have the helm suffix, instead they will be the exact same as for the package controller image, minus the `v` at the start of the tag?

Reply:
Yea I think that's what we're going for. With helm 3.8.1 we need to need the preceding  `v`, and the trailing `-helm`